src/create.js

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import {useNavigate} from "react-router-dom"
+
+
+
+const Create  = () => {
+
+const navigate = useNavigate();
+const [title, setTitle] = useState ("");
+const [body, setBody] = useState ("");
+const [author, setAuthor] = useState ("");
+const [pending, setPending] = useState(false);
+
+const handleSubmit  = (e) => {
+ e.preventDefault();
+ setPending(true)
+let newBlog = {title, body, author};
+
+setTimeout(()=>{
+    fetch("http://localhost:8000/blogs", {
+     method: "POST",
+    headers: {"content-Type": "application/json"},
+    body: JSON.stringify (newBlog)
+    })
+  .then (()=>{
+    setTitle("");
+    setBody("");
+    setAuthor("");
+    setPending(false);
+    navigate('/')
+  })  
+    
+}, 3000)
+
+
+// .then((res)=>{
+    // console.log(res)
+    // res.json()
+// console.log (newBlog)
+ 
+}
+
+
+
+    return ( 
+    <div className="create">
+
+<form onSubmit={handleSubmit}>
+
+<label>Title</label>
+<input type= "text" 
+required input
+onChange={(e)=>setTitle(e.target.value)}/>
```

Comment:
Please watch the video and check to confirm that something is wrong with this lines I selected. Please watch and re-watch until you find what is wrong. That is the only way you will be close to not making the same error again